lib/manager/npm/post-update/yarn.spec.ts

```diff
@@ -66,9 +67,33 @@ describe(getName(), () => {
       expect(fixSnapshots(execSnapshots)).toMatchSnapshot();
     }
   );
-  it.each([['1.22.0'], ['2.1.0']])(
+  it('only skips build if skipInstalls is false', async () => {
```

Comment:
Add newline between new tests. Can be deferred for existing tests.

Reply:
> Should also be tested on exiting yarn v1 and yarn v2 projects to be safe?

I tested the PR on Yarn 1 and Yarn 2 (https://github.com/ylemkimon/KaTeX).

> Can be deferred for existing tests.

What do you mean by 'deferred for existing tests'?

Reply:
Only add the newlines bewteen tests you are adding / changing. But feel free to do this for all in your modified spec files. 🙃